src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/instance_blackrock_spire.cpp

```diff
@@ -121,8 +121,9 @@ class instance_blackrock_spire : public InstanceMapScript
                 case NPC_SCARSHIELD_INFILTRATOR:
                     ScarshieldInfiltrator = creature->GetGUID();
                     break;
-                case NPC_FINKLE_EINHORN:
-                    creature->AI()->Talk(SAY_FINKLE_GANG);
+                case NPC_FINKLE_EINHORN: // @todo: move this either to SAI or CreatureAI
```

Comment:
@offl this might be some low hanging fruits for you

Reply:
dont